src/components/pages/Header.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./Header.css";
+import logoImg from "../../assets/images/logo/logo.svg";
+import profileImg from "../../assets/images/icons/ic_profile.svg";
+import { Link, useLocation } from "react-router-dom";
+
+function Header() {
+  const location = useLocation();
+  const currentPath = location.pathname;
+
+  return (
+    <header className="navbar">
+      <div className="nav-container">
+        <Link to="/" className="logoImg">
+          <img src={logoImg} alt="logo" />
+        </Link>
+        <Link
+          to="/freeboard"
+          className={`nav-link ${currentPath === "/freeboard" ? "active" : ""}`}
+        >
+          자유게시판
+        </Link>
+        <Link
+          to="/items"
+          className={`nav-link ${currentPath === "/items" ? "active" : ""}`}
+        >
```

Comment:
P2:
react-router-dom에서 쓰실만한 컴포넌트가 있을 것 같아 추천드립니다.
NavLink라는 친구로 공식문서에서의 설명은 아래와 같습니다.

`A <NavLink> is a special kind of <Link> that knows whether or not it is "active", "pending", or "transitioning". `
따라서 지금과 같이 active된 메뉴를 알고싶을때 사용하면 좋습니다.

https://reactrouter.com/en/main/components/nav-link

```suggestion
        <Link
          to="/items"
          className={({ isActive }) => `nav-link ${isActive ? 'active' : '' }`}
        >
```